locales/da.json

```diff
@@ -47,13 +47,13 @@
     "city": "By",
     "country": "Land",
     "postalCode": "Postnummer",
-    "province": "Region",
+    REMOVE: "province": "N/A (We dont use 'province' in Denmark)",
```

Comment:
This makes the JSON not valid. I think we should leave it as it was. Province field can be hidden in CSS if you don't need to show it in the form, but the locale file must remain valid JSON.